Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml

```diff
@@ -1127,6 +1133,13 @@ inputs:
   required: false
   description: Unique ID for the XDR alert.
   playbookInputQuery:
+- key: InternalIPRange
+  value: {}
+  required: false
+  description: |-
+    Please use "InternalIPRanges" input instead.
+    this input  deprecated.
```

Comment:
Probably should be "This (upper case) input is deprecated". @ShirleyDenkberg @evisochek

Reply:
Fixed @michalgold